libvips/iofuncs/operation.c

```diff
@@ -321,19 +316,18 @@ vips_operation_pspec_usage(VipsBuf *buf, GParamSpec *pspec)
 	/* These are the pspecs that vips uses that have interesting values.
 	 */
 	if (G_IS_PARAM_SPEC_ENUM(pspec)) {
-		GTypeClass *class = g_type_class_ref(type);
+		/* GParamSpecEnum holds a ref on the class so we just peek.
+		 */
+		GEnumClass *genum = g_type_class_peek(type);
 		GParamSpecEnum *pspec_enum = (GParamSpecEnum *) pspec;
 
-		GEnumClass *genum;
 		int i;
 
 		/* Should be impossible, no need to warn.
```

Comment:
Should we also update these "impossible" comments as `g_type_class_peek` can return null?